.github/workflows/ci.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         jdk: [8, 11]
     env:
       MVN_CONNECTION_OPTIONS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+      FLINK_URL: https://s3.amazonaws.com/flink-nightly/flink-1.16-SNAPSHOT-bin-scala_2.12.tgz
```

Comment:
this is actually super jank because there's 0 guarantees that this is compatible with the maven snapshot artifacts, but there's genuinely no better way at this time.

Reply:
Do we have a follow-up ticket for this?

Reply:
not yet; I don't know of a good way to solve it yet.